public/protocols/development.netcanvas/protocol.json

```diff
@@ -1351,6 +1419,47 @@
         }
       ]
     },
+    {
+      "id": "catBin1",
+      "type": "CategoricalBin",
+      "label": "Contact Types",
+      "subject": {
+        "entity": "node",
+        "type": "person"
+      },
+      "prompts": [
+        {
+          "id": "contactType",
+          "text": "What is the most frequent type of contact you have?",
+          "variable": "contactType",
+          "bucketSortOrder": [
+            {
+              "property": "nickname",
+              "direction": "desc"
+            }
+          ],
+          "binSortOrder": [
+            {
+              "property": "nickname",
+              "direction": "desc"
+            },
+            {
+              "property": "age",
+              "direction": "desc"
+            },
+            {
+              "property": "last_name",
+              "direction": "desc"
+            }
+          ]
+        },
+        {
+          "id": "discussType",
+          "text": "Discussion type variable, with a much longer prompt than the previous prompt. Please also note that this prompt uses the max number of bins.",
```

Comment:
Should we enforce a 'max number of bins' in the schema?

Reply:
I know Josh indicated that we would never want more than 8, so we could enforce that in the schema. I did not enforce a max in the code so there could be more, but the layout is designed with that max in mind. Eventually the layout will overflow (depending how wide your screen is), and the component is not scrollable, so that is good question -- should the code enforce that max? The schema? Or should the component just allow scrolling as a backup plan?

Reply:
If we want to enforce it, the schema should suffice. I opened #785 which can track this.